From d6c9765de3ce19bec9b67cb6cbba8827240565b1 Mon Sep 17 00:00:00 2001
From: Russ Weight <russell.h.weight@intel.com>
Date: Thu, 1 Apr 2021 14:43:28 -0700
Subject: [PATCH 47/93] net: ethernet: silicom: Fix N5010_PHY dependency

The N5010_PHY config option has a dependency on MFD_INTEL_M10_BMC which
has been renamed to MFD_INTEL_M10_BMC_CORE.

Signed-off-by: Russ Weight <russell.h.weight@intel.com>
---
 drivers/net/ethernet/silicom/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/silicom/Kconfig b/drivers/net/ethernet/silicom/Kconfig
index 2366418f5f25..204e1d97e15a 100644
--- a/drivers/net/ethernet/silicom/Kconfig
+++ b/drivers/net/ethernet/silicom/Kconfig
@@ -18,7 +18,7 @@ if NET_VENDOR_SILICOM
 
 config N5010_PHY
 	tristate "Fixed Phy Driver for Silicom PAC N5010"
-	depends on MFD_INTEL_M10_BMC && FIXED_PHY
+	depends on MFD_INTEL_M10_BMC_CORE && FIXED_PHY
 	help
 	  The n5010 card consists of a primary FPGA running the network controller,
 	  and a side-fpga running the board management controller (bmc). This driver
-- 
2.39.0

